packages/frontend/core/src/modules/pdf/views/components.tsx

```diff
@@ -0,0 +1,185 @@
+import { Scrollable } from '@affine/component';
```

Comment:
these components are actually related with pdf-viewer and nothing todo with pdf page.
move to viewer folder

Reply:
Because there will be pdf embed view next, I expect to reuse these, so I put them here.

Reply:
reasonable